protocol/x/revshare/types/msg_update_unconditional_revshare_config.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/lib"
+)
+
+var _ types.Msg = &MsgUpdateUnconditionalRevShareConfig{}
+
+// ValidateBasic performs validation to check the total percentage
+// across all configs is <= 100
+func (msg *MsgUpdateUnconditionalRevShareConfig) ValidateBasic() error {
+	var totalRevsharePercentagePpm uint32 = 0
+	config := msg.Config
+	for _, config := range config.Configs {
+		totalRevsharePercentagePpm += config.SharePpm
+	}
+
+	if totalRevsharePercentagePpm > lib.OneMillion {
```

Comment:
Practically won't hit this, but let's make this less permissive: 
```suggestion
	if totalRevsharePercentagePpm >= lib.OneMillion {
```